### PR TITLE
Deploy app

### PR DESCRIPTION
**Pull request by kelibst**
Deployed to Heroku

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Nice Try
I was not able to see how your app works because the live link doesn't work. Please make the following changes.
- [ ] The live link points to the odin project page. Fix that
- [ ] Make your Readme more professional. Check my comments


### Martin Afani

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 ## Live Demo
 
+# ([Live link](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))
```
Comment (whiz25) [inline README.md]: - fix this link

File: README.md [hunk 2]
```diff
@@ -37,6 +39,7 @@ Setup database with:
    rails db:migrate
 ```
 
+## Live link
```
Comment (whiz25) [inline README.md]: - What does TBA mean? Is it an instruction for deploying the app to heroku? Please give us precise details on how one can deploy this app to heroku

**Review by whiz25 — APPROVED** [review]
## Congratulations :clap:
You made the required changes and its therefore a :+1: from me. Merge and proceed.

### Martin Afani

**Review by whiz25 — APPROVED** [review]
## Hey @kelibst 
You have made the required changes for this milestone. I must admit making a mistake in approving milestone 7 of this project. It's in that milestone that you are supposed to enable signup using facebook. 
Please add that feature as well. Meanwhile, I will approve this PR since it meets the minimum requirements.

### Martin Afani

**Comment by kelibst** [comment]
Thanks guys

**Pull request merged**